python/ui/method/add_remove_page.py

```diff
@@ -0,0 +1,23 @@
+from python.common.baseui import BasicFunctions
+from selenium.webdriver.common.by import By
+
+
+class AddRemove(BasicFunctions):
+    add_remove_header_text = (By.XPATH, '//*[@id="content"]/h3')
+    add_element_btn = (By.XPATH, '//*[@id="content"]/div/button')
+    delete_element_btn = (By.XPATH, '//*[@id="elements"]/button')
+
+    def text_is_present(self):
+        self.verify_is_present(self.add_remove_header_text)
+
+    def add_element_button_clickable(self):
+        self.mouse_click(self.add_element_btn)
+
+    def delete_element_button_clickable(self):
+        self.mouse_click(self.delete_element_btn)
+
+    def delete_all_delete_btns(self):
```

Comment:
This is super nitpicky, but I would try to be consistent with shortening or going full length for words. I would adjust `btns` to `buttons` here since the previous two are `button`

Reply:
then again you went with `btn` in your selector variables. So it might make more sense to change the `button` in your methods to `btn`

Reply:
Good catch.
I think this occurred because we swapped from KP -> Trent midway through code pairing.